Client/Extensions/UIPasteboardExtensions.swift

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import UIKit
+
```

Comment:
You'll now need to add `#import MobileCoreServices` to the top here. So sorry for taking so long to get round to this.